src/Spinners/index.ts

```diff
@@ -0,0 +1 @@
+export { default as LoadSpinner } from './LoadSpinner';
```

Comment:
If we go the route of using an `enum` for the color values above, that will also need to be exported from the library in here

Reply:
Also, I just realized that we'll need to export these component from the library itself by adding `export * from './Spinners'` to the `src/index.ts` file.

Reply:
I needed to export enum from the LoadSpinnerText text that I used in the LoadSpinner. Do I need to exported in src/Spinners/index.ts ?

Reply:
This is correct here -- we need to export the `SPINNER_TEXT` enum, but not the `LoadSpinnerText`. This `src/Spinners/index.ts` is what gets exported from the main `src/index.ts`, so only these two exports will be available from the main application; the `LoadSpinnerText` itself will not be exported publicly.